src/aentity-components.js

```diff
@@ -0,0 +1,3 @@
+export default function registerComponents() {
```

Comment:
Title of this file should include "gtlf" somewhere, maybe `gltf-component-mappings.js` or `a-gltf-entity-components.js`.

Reply:
Also this file should technically require `a-gltf-entity.js`